README.md

```diff
@@ -68,69 +68,73 @@ should be used. Older releases are no longer supported.
 
-|                   |LVM                    |direct              |
+|                   |`LVM`                    |`direct`              |
 |:--                |:--                    |:--                 |
 |Main advantage     |avoids free space fragmentation<sup>1</sup>   |simpler, somewhat faster, but free space may get fragmented<sup>1</sup>   |
 |What is served     |LVM logical volume     |pmem block device   |
 |Region affinity<sup>2</sup>    |yes: one LVM volume group is created per region, and a volume has to be in one volume group  |yes: namespace can belong to one region only  |
 |Startup            |two extra stages: pmem-ns-init (creates namespaces), vgm (creates volume groups)   |no extra steps at startup |
-|Namespace modes    |*fsdax* mode<sup>3</sup> namespaces pre-created as pools   |namespace in `fsdax` mode created directly, no need to pre-create pools   |
+|Namespace modes    |`fsdax` mode<sup>3</sup> namespaces pre-created as pools   |namespace in `fsdax` mode created directly, no need to pre-create pools   |
 |Limiting space usage | can leave part of device unused during pools creation  |no limits, creates namespaces on device until runs out of space  |
 | *Name* field in namespace | *Name* gets set to 'pmem-csi' to achieve own vs. foreign marking | *Name* gets set to VolumeID, without attempting own vs. foreign marking  |
 |Minimum volume size| 4 MB                   | 1 GB (see also alignment adjustment below) |
 |Alignment requirements |LVM creation aligns size up to next 4MB boundary  |driver aligns  size up to next alignment boundary. The default alignment step is 1 GB. Device(s) in interleaved mode will require larger minimum as size has to be at least one alignment step. The possibly bigger alignment step is calculated as interleave-set-size multiplied by 1 GB |
 
-<sup>1 </sup> *Fragmented free space* state may develop via series of creation and deletion operations where the driver is no longer able to allocate a new namespace contiguously, although the free capacity (i.e. sum of available free section sizes) indicates the opposite. The PMEM-CSI driver is not capable of de-fragmenting the space or create allocation using combined smaller blocks. 
-A simplified example: This state develops after steps: create 63 GB namespace, create 1 GB namespace, delete 63 GB namespace. The free capacity is 127 GB, but the driver fails to create a namespace bigger than 64 GB. 
+<sup>1 </sup> **Free space fragmentation** is a problem when there appears to be
+enough free capacity for a new namespace, but there isn't a contiguous region
+big enough to allocate it. The PMEM-CSI driver is only capable of allocating
+continguous memory to a namespace and cannot de-fragment or combine smaller
+blocks. For example, this could happen when you create a 63 GB namespace,
+follwed by a 1 GB namespace, and then delete the 63 GB namespace. Eventhough
+there is 127 GB available, the driver cannot create a namespace larger than
+64 GB. 
```

Comment:
This can be enhanced further: the "there is 127 GB available" is stated without explaining from what that follows, i.e. the total capacity is missing. What about "this could happen when you create a 63 GB namespace in a region of 128 GB, followed by a 1 GB namespace, ..."?

"follwed" must be "followed". Funny how many different people must have read that already without noticing :joy:

Reply:
> Funny how many different people must have read that

High chance you are the first one reading this (new text added in this PR) in detail. And even for published version, I dare not hope for having many readers.

Reply:
> High chance you are the first one reading this (new text added in this PR)

Oh, right. I thought it had been already in the branch.

> And even for published version, I dare not hope for having many readers.

What? You don't want people to use PMEM-CSI?! :scream:

Reply:
let me rephrase, I hope,want for many but I estimate number is low despite our hopes.

Reply:
Good catch. Thanks.